src/communication/network_interface_cl.c

```diff
@@ -7062,8 +7082,8 @@ qfile_get_list_file_page (QUERY_ID query_id, VOLID volid, PAGEID pageid, char *b
  *
  * NOTE: If xasl_header_p is not null, also XASL node header will be requested
  */
-XASL_ID *
-qmgr_prepare_query (COMPILE_CONTEXT * context, XASL_STREAM * stream, const OID * user_oid)
+int
+qmgr_prepare_query (COMPILE_CONTEXT * context, XASL_STREAM * stream)
```

Comment:
@ribram 

It would be great if you can update the function comment. 

```C
 * Send XASL stream and receive XASL file id (XASL_ID) as a result.
 * If xasl_buffer == NULL, the server will look up the XASL cache and then
 * return the cached XASL file id if found, otherwise return NULL.
 * This function is a counter part to sqmgr_prepare_query().
 *
 * return: XASL_ID
```

```C
 *   user_oid(in):
```